samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -348,24 +353,24 @@ static void fmdn_provisioning_state_changed(bool provisioned)
 
 	/* Triggered on the unprovisioning operation. */
 	if (factory_reset_executed) {
-		LOG_INF("Please press a button to put the device in the Fast Pair discoverable "
```

Comment:
You can leave this log here. This is information on how to trigger another FMDN provisioning (you need explicit user action).